### DISCUSSION TITLE
How to  enter low-power mode? and how to wakeup?

### DISCUSSION THREAD

**Discussion by liuquanli1970** [stm32duino/Arduino_Core_STM32#2470] [General]
In order to better utilize electrical energy, STM32 adopts intermittent operation. Excuse me, what if I enter low-power mode (sleep mode)?. How to wake up externally after hibernation?

**Comment by fpistm** [comment]
Hi @liuquanli1970 
You have several examples here: https://github.com/stm32duino/STM32LowPower/tree/main/examples
